Release `v0.1.3`

rudyflores opened the pull request:
**What It Does**
<!-- A list of relevant issues, enhancements, fixed bugs, etc -->

Prepare release `v0.1.3` with much needed vulnerability fixes

**How to Test**
<!-- If a bug has been fixed, how can reviewers verify that the change(s) fixed it? -->

**Review Checklist**
I certify that I have:
- [ ] tested my changes
- [ ] added/updated automated tests
- [ ] updated the changelog
- [ ] followed the [contribution guidelines](https://github.com/zowe/zowe-cli/blob/master/CONTRIBUTING.md)


**Additional Comments**
<!-- Anything else noteworthy about this pull request. This section is optional. -->

rudyflores commented:
@t1m0thyj do you know why the changelog check keeps failing? I formatted the changelog to look similar to how the changelog looks in ZE but it seems to still fail 🤔

traeok commented:
> do you know why the changelog check keeps failing? I formatted the changelog to look similar to how the changelog looks in ZE but it seems to still fail 🤔

Could it be that the changelog check expects `## Recent Changes` instead of `## TBD Release` for the header? I noticed that the check printed an error about a missing header, but I wasn't sure if that was the header causing issues.

rudyflores commented:
@t1m0thyj @traeok thanks for the help! I added as well a changelog the `vscode` package and added the fix for the error #15 that Timothy created a while back and was addressed in this new version :)

t1m0thyj reviewed (approved):
LGTM, thanks @rudyflores!

traeok reviewed (approved):
LGTM, thanks @rudyflores 😁
